all/pom.xml

```diff
@@ -172,25 +172,19 @@
         <dependency>
             <groupId>com.wttech.aem.acm</groupId>
             <artifactId>acm.ui.apps</artifactId>
-            <version>${project.version}</version>
+            <version>${revision}</version>
```

Comment:
still ${project.version} ; 

sry, please revert it; in depedencies ${revision} should not be used - https://maven.apache.org/guides/mini/guide-maven-ci-friendly.html